README.md

```diff
@@ -15,7 +15,7 @@ This package was designed to work with [Worker Sites](https://workers.cloudflare
 
 ### `getAssetFromKV`
 
-`getAssetFromKV` is a function that takes a `FetchEvent` object and returns a `Response` object if the request matches an asset in KV, otherwise it will throw an `Error`.
+`getAssetFromKV` is a function that takes a `FetchEvent` object and returns a `Response` object if the request matches an asset in KV, otherwise it will throw an `KVError`.
```

Comment:
```suggestion
`getAssetFromKV` is a function that takes a `FetchEvent` object and returns a `Response` object if the request matches an asset in KV, otherwise it will throw a `KVError`.
```

Reply:
i think it is worth calling out the different kv error types, as well.